dev-cpp/blitz/blitz-1.0.3_pre20200524.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR=emake
```

Comment:
We really need this? :(

Reply:
Usually it requires if Fortran code is compiled. I.e. Sundials had compilation problems with ninja.

In blitz++ the Fortran compiler is required only optionally to build fortran benckmark that is used to compare speed of blitz++ with pure fortran array operations. 

In this ebuild this benchmark isn't built so maybe blitz++ could use ninja. I need to recheck it.